apps/console/src/features/identity-providers/components/settings/attribute-management/uri-attributes-settings.tsx

```diff
@@ -46,6 +46,10 @@ interface AdvanceAttributeSettingsPropsInterface extends TestableComponentInterf
      * Specifies if the component should only be read-only.
      */
     isReadOnly: boolean;
+        /**
```

Comment:
There is a formatting issue here

Reply:
Fixed with https://github.com/wso2/identity-apps/pull/3279/commits/0211cfdea4d21643b59fe812ddcab30928cb0fe5